docs/user-guide/rs.rst

```diff
@@ -0,0 +1,230 @@
+.. include:: /content/nemo.rsts
+
+.. _model-aligner-rs:
+
+Model Alignment by Rejection Sampling
+@@@@@@@@@@@@@@@@@@@@@@@
```

Comment:
extend the @ characters so they match the length of the title

Model Alignment by Rejection Sampling
@@@@@@@@@@@@@@@@@@@

Reply:
Extended the @ characters.